oggm_edu/glacier.py

```diff
@@ -203,8 +217,10 @@ def _to_json(self):
 
     def copy(self):
         """Return a copy of the glacier. Useful for quickly creating
-        new glaciers."""
-        return copy.deepcopy(self)
+        new glaciers. It does assign a new id to the glacier."""
+        copied_glacier = copy.deepcopy(self)
+        copied_glacier.id = str(uuid.uuid4())[-5:]
```

Comment:
`copy` should have an `id` kwarg in case the user doesn't want an obscure id to be created automatically.

Reply:
Agree. `GlacierCollection.fill(...)` should then also have an optional like `ids` (list of ids for the copies).

Reply:
See https://github.com/OGGM/oggm-edu/pull/188/commits/94a7db593174ea17b31117e04849f1ac06741890